docs/dev/news/README.md

```diff
@@ -0,0 +1,15 @@
+# Latest Dafny changes
+
+Each file in this folder describes one change in Dafny since the latest release.
+
+Files in this directory are named `<PR number>.<kind>` and each contains release notes for one merged PR.  `kind` is `feat` or `fix`.  No need to include a link to the PR in the file: it will be added automatically.
+
+## Examples
+
+`1234.fix`
+
+```
+Improved toaster settings prevents Dafny from ever burning toast. ??
+```
+
+At release time, this becomes a bullet point in the "Bug fixes" section of `RELEASE_NOTES.md` with a link to <https://github.com/dafny-lang/dafny/pull/1234>.
```

Comment:
Can you exactly say how it will be included? I assume that for the above, it will automatically add the line

- fix: Improved toaster settings prevents Dafny from ever burning  (https://github.com/dafny-lang/dafny/pull/1234)

Reply:
It actually becomes 

```markdown
## Bug fixes

- Improved toaster settings prevent Dafny from ever burning toast.
  (https://github.com/dafny-lang/dafny/pull/1234)
```

I don't think the README should say this, though.

Reply:
I agree this is an implementation detail, but it's a detail important enough so that I seriously think it's worth mentioning it.
Otherwise, we might get developers down the road who will want to prefix their subjects with "fix:" or "feat:" or want to write something on two lines, or they will want to include a custom URL, or they won't know if they have to put a capital or not on the first letter, or if they need to include a space at the end.... all of this because the transformation is only `provide`d but not `reveal`ed :-)

And we should get rid of the need to add "??". By default, the URL should be appended automatically. I don't see a clear use case where a developer would want to include this link anywhere in the subject link.

Reply:
OK, done.  I also added a test to make sure that the format was correct.

> And we should get rid of the need to add "??". By default, the URL should be appended automatically. 

`??` was a typo, you spotted it and I already accepted your patch.  Did I miss something else?

Reply:
Oh I probably missed your fix. Thanks :-)